www/pages/working-with-qgis-plugin.md

```diff
@@ -26,16 +26,44 @@ To install Mergin plugin in QGIS:
 
 	**Note:** if you cannot see your Browser panel, you can enable it by going to the QGIS main menu > View > Panels > Browser
 
+5. A new toolbar should also appear in your QGIS toolbars:
+
+<p align="center"><img src="../images/qgis-plugin/plugin_toolbar.png"></p>
+
 After installation, you need to configure the plugin by entering your Mergin username and password:
 
 1. From the QGIS Browser panel, right-click on **Mergin** and select **Configure**
+		(alternatively from the Mergin toolbar, select the configure icon )
 
 2. In the new window, type in your username (or email address used for signing up with Mergin) and password
 
 	<p align="center"><img src="../images/qgis-plugin/configure.png" width="50%"></p>
 
 You can select to store the password. For that you need to have already configured your [QGIS password manager](https://docs.qgis.org/3.10/en/docs/user_manual/auth_system/auth_overview.html?highlight=password#master-password)
 
+### Creating a new Mergin project
+
+To create a new project, you can start from a blank project or alternatively open an existing project. You can then select **Create Mergin Project** from the toolbar:
+
+![Project creation icon in toolbar](https://www.lutraconsulting.co.uk/img/posts/mergin_plugin_project_wizard_1.png)

```

Comment:
It would be good to copy the images to this repo and link to the copies, so that we can update them easily when needed and do not depend on the lutra website.